keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -475,24 +490,50 @@ coverage of interactions in these dimensions:
-| 'string' with format=datetime           | timestamp (protobuf.Timestamp) |
-
-xref: [CEL types](https://github.com/google/cel-spec/blob/master/doc/langdef.md#values), [OpenAPI types](https://swagger.io/specification/#data-types).
+| OpenAPIv3 type                                     | CEL type                                                                                                       |
+| -------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| 'object' with Properties                           | object / "message type"                                                                                        |
+| 'object' with AdditionalProperties                 | map                                                                                                            |
+| 'object' with x-kubernetes-embedded-type           | <treatment is the same as 'object' more details below>                                                         |
+| 'object' with x-kubernetes-preserve-unknown-fields | <treatment is the same as 'object', more details below>                                                        |
+| x-kubernetes-embedded-int-or-string                | object with 'intVal' (type: int) and 'strVal' (type: string) fields                                            |
+| 'array                                             | list                                                                                                           |
+| 'array' with x-kubernetes-list-type=map            | list with map based Equality & unique key guarantees                                                           |
+| 'array' with x-kubernetes-list-type=set            | list with set based Equality & unique entry guarantees                                                         |
+| 'boolean'                                          | boolean                                                                                                        |
+| 'number' (all formats)                             | double                                                                                                         |
+| 'integer' (all formats)                            | int (64)                                                                                                       |
+| 'null'                                             | null_type                                                                                                      |
+| 'string'                                           | string                                                                                                         |
+| 'string' with format=byte                          | string (containing base64 encoded data)                                                                        |
```

Comment:
why is this different than `'string' with format=binary`? also, I don't see format "binary" in the openapi validation

Reply:
This is an invalid case. I had seen 'binary' defined in the OpenAPIv3 spec and didn't realize we don't support it on CRDs. I'll drop this case.

I've updated the code to map the string 'byte' format to the CEL byte type.